### PR TITLE
Feat/Intune/add_Smart_description

### PR DESCRIPTION
**Pull request by Thibault-MAL**
Add some generic smart description on Intune

### CODE REVIEW

File: Microsoft/microsoft-intune/_meta/smart-descriptions.json [hunk 1]
```diff
@@ -1,3 +1,66 @@
-{
-
-}
\ No newline at end of file
+[  {
+    "value": "Operation {action.name} made on device {host.name}",
```
Comment (PierrickV) [inline Microsoft/microsoft-intune/_meta/smart-descriptions.json:2]:
```suggestion
    "value": "Operation {action.name} preformed on device {host.name}",
```

**Review by squioc — DISMISSED** [review]

File: Microsoft/microsoft-intune/_meta/smart-descriptions.json [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+      },
+      {
+        "field": "action.name"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "host.name",
+        "target": "action.name",
+        "type": "made"
```
Comment (PierrickV) [inline Microsoft/microsoft-intune/_meta/smart-descriptions.json:15]:
```suggestion
        "type": "performed operation"
```

File: Microsoft/microsoft-intune/_meta/smart-descriptions.json [hunk 3]
```diff
@@ -1,3 +1,66 @@
...
+      }
+    ]
+  },
+  {
+    "value": "Operation {action.name} made by {user.name}",
```
Comment (PierrickV) [inline Microsoft/microsoft-intune/_meta/smart-descriptions.json:20]:
```suggestion
    "value": "Operation {action.name} performed by {user.name}",
```

File: Microsoft/microsoft-intune/_meta/smart-descriptions.json [hunk 4]
```diff
@@ -1,3 +1,66 @@
...
+      },
+      {
+        "field": "action.name"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "user.name",
+        "target": "action.name",
+        "type": "made"
```
Comment (PierrickV) [inline Microsoft/microsoft-intune/_meta/smart-descriptions.json:33]:
```suggestion
        "type": "performed operation"
```

File: Microsoft/microsoft-intune/_meta/smart-descriptions.json [hunk 5]
```diff
@@ -1,3 +1,66 @@
...
+      }
+    ]
+  },
+    {
+    "value": "{user.name} on device {host.name}",
```
Comment (PierrickV) [inline Microsoft/microsoft-intune/_meta/smart-descriptions.json:38]:
```suggestion
    "value": "{user.name} logged on device {host.name}",
```

File: Microsoft/microsoft-intune/_meta/smart-descriptions.json [hunk 6]
```diff
@@ -1,3 +1,66 @@
...
+    "relationships": [
+      {
+        "source": "user.name",
+        "target": "host.name",
+        "type": "logged on"
+      }
+    ]
+  },
+  {
+    "value": "{host.name} compliant status is {intune.compliant}",
```
Comment (PierrickV) [inline Microsoft/microsoft-intune/_meta/smart-descriptions.json:56]:
```suggestion
    "value": "{host.name} compliance status evaluated at {intune.compliant}",
```

I would have say compliance :thinking:

**Comment by PierrickV** [comment]
Tests files must be updated

**Review by PierrickV — COMMENTED** [review]
Tests files must be updated

File: Microsoft/microsoft-intune/ingest/parser.yml [hunk 7]
```diff
@@ -27,7 +27,7 @@ stages:
           host.mac: '{{json_event.message.properties.WifiMacAddress}}'
           host.name: '{{json_event.message.properties.ManagedDeviceName or json_event.message.properties.DeviceHostName }}'
           host.type: '{{json_event.message.properties.Model}}'
-          intune.compliant: '{{json_event.message.properties.CompliantState}}'
+          microsoft.intune.compliant: '{{json_event.message.properties.CompliantState}}'
```
Comment (PierrickV) [inline Microsoft/microsoft-intune/ingest/parser.yml:30]: I think that in vendor fields, we should stay as close as possible from the original field name (here `properties.CompliantState`)

Maybe @squioc as a different point of view ?

Comment (squioc) [reply Microsoft/microsoft-intune/ingest/parser.yml:30]: So, @pve you suggest `microsoft.intune.compliant_state`?

**Review by squioc — APPROVED** [review]

**Comment by PierrickV** [comment]
@squ yes I think that it would be better

**Pull request merged**